README.md

```diff
@@ -52,14 +52,13 @@ To validate your downloaded file's integrity, compare the lines in checksum.txt
 
 ### Install / Build
 
-Clone the repo, and use a simple `npm install`.  You can then use a `npm run
-build` to create an optimised static build (serve it with e.g. [serve][serv]).
+Clone the repo, and use a simple `npm install`. You can then use a `npm run build` to create an optimised static build (serve it with e.g. [serve][serv]).
```

Comment:
I think you accidentally linted the readme?

Reply:
fixed!